### ISSUE TITLE
gdbserver path was changed in binutils-gdb

### ISSUE THREAD

**Issue by falaleevms** [crosstool-ng/crosstool-ng#1341]
- crosstool-NG version 1.24.0.6-afaf7b9
- CentOS Linux release 7.3.1611
- [defconfig.txt](https://github.com/crosstool-ng/crosstool-ng/files/4659357/defconfig.txt), [config.txt](https://github.com/crosstool-ng/crosstool-ng/files/4659359/config.txt)
- [build.log](https://github.com/crosstool-ng/crosstool-ng/files/4659351/build.log) 

Following commit in binutils-gdb repository changes location of gdbserver:
[Move gdbserver to top level
](http://sourceware.org/git/?p=binutils-gdb.git;a=commit;h=919adfe8409211c726c1d05b47ca59890ee648f1) (not released yet, but in master since 7 Feb 2020)

So building gdbserver expectedly fails with following error:
```
 chmod: cannot access '/SCRATCH/falaleev/git/crosstool-ng/.build/arc-multilib-elf/src/gdb/gdb/gdbserver/configure': No such file or directory
```

**Comment by abrodkin** [comment]
I guess something along that lines of that change might fix it: https://github.com/abrodkin/crosstool-ng/commit/2f9b431228c58fe98704a72390672942935ac8e3.

So once GDB 10.1 is released we may get back to it and add support of that new special handling for newer versions.

**Comment by cpackham** [comment]
Yeah I think this will have to wait until there's an official release. We'll still want to maintain support for the older versions so we'll need a version of @abrodkin's patch that distinguishes between gdb < 10 and newer versions.

**Comment by abrodkin** [comment]
@cpackham what stops me from implementation of the proper improvement (handling of both pre-10 & 10+ GDB versions) is absence of the config symbol I may check for. Like `CT_GDB_10_1_or_later`.

**Comment by cpackham** [comment]
Resolved in #1573
